### ISSUE TITLE
[FEATURE] please improve the killaura Hurttime

### ISSUE THREAD

**Issue by apexCode1337** [CCBlueX/LiquidBounce1.8-Issues#3510]
please add a killaura option that keeps aiming and blocking at a target even if the hurttime is set to 0 and the target is higher

**Comment by yorik100** [comment]
And add an option to unblock while hitting

**Comment by yorik100** [comment]
Bump

**Comment by apexCode1337** [comment]
Hm?

**Comment by yorik100** [comment]
> Hm?

Just posting there to remind it's important to fix it

**Comment by apexCode1337** [comment]
Yes it is😕

**Comment by apexCode1337** [comment]
Btw what do you mean with unblock while hitting?

**Comment by yorik100** [comment]
> Btw what do you mean with unblock while hitting?

Unblock when attacking during the attack tick

**Comment by apexCode1337** [comment]
Isn't it like that already?

**Comment by yorik100** [comment]
> Isn't it like that already?

No, it unblocks attack then block which is detected af on a lot of servers

**Comment by apexCode1337** [comment]
What blocks more damage?

**Comment by yorik100** [comment]
Bump

**Comment by apexCode1337** [comment]
I've been waiting for it xd
